code/datums/diseases/transformation.dm

```diff
@@ -45,13 +45,10 @@
 		if(affected_mob.notransform)
 			return
 		affected_mob.notransform = 1
-		for(var/obj/item/W in affected_mob)
-			if(istype(W, /obj/item/weapon/implant))
-				qdel(W)
-				continue
-			W.layer = initial(W.layer)
-			W.loc = affected_mob.loc
-			W.dropped(affected_mob)

```

Comment:
Those last three lines were basically copypasta from unEquip() code.
The code is also dropping chest cavity items and dental implants on the ground, which probably shouldn't happen.